readme.md

```diff
@@ -1,3 +0,0 @@
-# Bean Validation API
```

Comment:
it has been renamed to README.md and augmented (and too many differences for git to see it's a move)